app/src/main/java/com/mho/training/sources/PersonDataSource.kt

```diff
@@ -17,24 +17,24 @@ import java.io.IOException
 
 class PersonDataSource(private val resources: Resources) : RemotePersonDataSource {
 
-    override suspend fun getPerson(personId: Int): Result<Person> = withContext(Dispatchers.IO) {
+    override suspend fun getPerson(personId: Int): DataResult<Person> = withContext(Dispatchers.IO) {
         safeApiCall(
             call = { requestPerson(personId) },
             errorMessage = resources.getString(R.string.error_unable_to_fetch_person)
         )
     }
 
-    private suspend fun requestPerson(movieId: Int): Result<Person> {
+    private suspend fun requestPerson(movieId: Int): DataResult<Person> {
```

Comment:
Why are you wating for a movieId when you request a person?

Reply:
Typo! Got it!